test/wsdl-test.js

```diff
@@ -101,6 +101,24 @@ wsdlStrictTests['should handle type ref'] = function(done) {
   });
 };
 
+wsdlStrictTests['should get empty namespace prefix'] = function(done) {
+  var expectedMsg = '<ns1:fooRq xmlns:ns1="http://example.com/bar/xsd"' +
+    ' xmlns="http://example.com/bar/xsd"><bar1:paymentRq' +
+    ' xmlns:bar1="http://example.com/bar1/xsd">' +
+    '<bar1:bankSvcRq>' +
+    '<requestUID>001</requestUID></bar1:bankSvcRq>' +
+    '</bar1:paymentRq></ns1:fooRq>';
+  // var expectedMsg = 'gg';
+
+  soap.createClient(__dirname + '/wsdl/elementref/foo.wsdl', {strict: true}, function(err, client) {
```

Comment:
It is quite difficult to understand for me to writing test codes in test/request-response-samples. can you help me out?